data.sql

```diff
@@ -12,3 +12,26 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Boarmon', '2005-06-07', 20.4, true, 7);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', 17, true, 3);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Ditto', '2022-03-14', 22, true, 4);
+
+INSERT INTO owners (full_name, age)
+  VALUES  ('Sam Smith', 34),
+          ('Jennifer Orwell', 19),
+          ('Bob', 45),
+          ('Melody Pond', 75),
+          ('Dean Winchester', 14),
+          ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES  ('Pokemon'), ('Digimon');
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE name = 'Agumon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') WHERE name in ('Pikachu', 'Gabumon');
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name in ('Devimon', 'Plantmon');
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name in ('Charmander', 'Squirtle', 'Blossom');
```

Comment:
- I noticed that there's a little mix-up between the animals owned by Bob and Melody Pond. Kindly check the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md) to see the correct animals owned by both. 😉 

Below 👇 is a screenshot of the requirement.

![](https://i.imgur.com/zYfN25r.png)